Contribute/how-to-write-docs-auth-pack.md

```diff
@@ -0,0 +1,89 @@
+|Link         |`selectLinkType`      |Inserts a link. When you select this command, the following sub-menu appears.<br><br>`External`: Link to a web page by URI. Must include "http" or "https".<br>`Internal`: Insert a relative link to another file in the same repo. After selecting this option, type in the command window to filter files by name, then select the file you want. <br>`Bookmark in this file`: Choose from a list of headings in the current file to insert a properly formatted bookmark.<br>`Bookmark in another file`: First, filter by file name and select the file to link to, then choose the appropriate heading within the selected file.|
+|Image        |`insertImage`     |Type alternate text (required for accessibility) and select it, then call this command to filter the list of supported image files in the repo and select the one you want. If you haven't selected alt text when you call this command, you will be prompted for it before you can select an image file.|
+|Include      |`insertInclude`   |Find a file to embed in the current file.|
+|Snippet      |`insertSnippet`   |Find a code snippet in the repo to embed in the current file.|
+|Video        |`insertVideo`     |Add an embedded video.|
+|Preview      |`previewTopic`    |Preview the active topic in a side-by-side window using the DocFX extension.  If the DocFX extension is not installed or is installed but disabled, the topic will not render.
+
+## How to assign keyboard shortcuts
+
+1. Type `CTRL+K` then `CTRL+S` to open the Keyboard Shortcuts list.
+1. Search for the command, such as `formatBold`, for which you want to create a custom keybinding.
+1. Click the plus that appears near the command name when you mouse over the line.
+1. After a new input box is visible, type the keyboard shortcut you want to bind to that particular command. For example, to use the common shortcut for bold, type `ctrl+b`.
+1. It's a good idea to insert a `when` clause into your keybinding, so it won't be available in files other than Markdown. To do this, open keybindings.json and insert the following line below the command name (be sure to add a comma between lines):
+   
+    `"when": "editorTextFocus && editorLangId == 'markdown'"`
+
+    Your completed custom keybinding should look like this in keybindings.json:
+
+    ```json
+    // Place your key bindings in this file to overwrite the defaults
+    [
+        {
+            "key": "ctrl+b",
+            "command": "formatBold",
+            "when": "editorTextFocus && editorLangId == 'markdown'"
+        }
+    ]
+    ```
+
+1. Save keybindings.json.
+
+See [Keybindings](https://code.visualstudio.com/docs/getstarted/keybindings) in the VS Code docs for more information.
+
+## How to show the legacy "Gauntlet" toolbar
```

Comment:
Gauntlet was never released publicly. Do we want to re-phrase some of this to avoid the use of a code name that many readers will not be familiar with? 

Or, if our long term plan is to deprecate the toolbar, should this section be moved to an internal document? /cc @Jim-Parker

Reply:
We should remove reference to the Gauntlet term, and just explain that there is an optional toolbar that can be turned on. We should also be more specific that the setting in question is "markdown.showToolbar": true ... instead of the "the desired setting"